py/examples/combobox.py

```diff
@@ -24,6 +25,8 @@ async def serve(q: Q):
                         choices=combobox_choices, disabled=True),
             ui.combobox(name='combobox_error', label='Enter or choose a color', placeholder='Color...', value='Blue',
                         choices=combobox_choices, error='This combobox has an error!'),
+            ui.combobox(name='combobox_trigger', trigger=True, label='Enter or choose a color (trigger on selection)', placeholder='Color...', value=q.args.combobox_trigger,
```

Comment:
@mturoci is there something wrong with using the value that was recently send to the server? And how do I conditionally pass arguments in Python like in JS `value || fallback`?

Reply:
It's confusing to read. It's enough if the example line demonstrates how to make a combobox submit its value when changed. The `value=q.args.combobox_trigger` is unnecessary.

Reply:
>is there something wrong with using the value that was recently send to the server

`value` is meant to be "initial" value - all wave components should be uncontrolled. That means the `value` would work only if you recreate the card.

>how do I conditionally pass arguments in Python 

`value or fallback`

Reply:
Anyway it's better to create a separate example for `trigger`, like we have for `textbox` for example. The reason is that triggering will submit also filled values from other comboboxes.

Reply:
I added a new example like we have for textbox